File: abc/abc249/b.py
# https://atcoder.jp/contests/abc249/submissions/38036192

s = input()
n = len(s)
big = False
small = False
diff = True

for i in range(n):
  for j in range(i + 1, n):
    if s[i] == s[j]:
      diff = False

    if s[i] <= 'Z':
      big = True

    if 'a' <= s[i]:
      small = True

    if s[j] <= 'Z':
      big = True

    if 'a' <= s[j]:
      small = True

if big and small and diff:
  print("Yes")
else:
  print("No")

File: abc/abc249/b.java
// https://atcoder.jp/contests/abc249/submissions/47099213

import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    String s = stdIn.next();
    int n = s.length();

    boolean check = false;
    for(int i = 0; i < n; i++) {
      if(s.charAt(i) < 'a') {
        check = true;
        break;
      }
    }

    if(!check) {
      System.out.println("No");
      System.exit(0);
    }

    check = false;
    for(int i = 0; i < n; i++) {
      if('Z' < s.charAt(i)) {
        check = true;
        break;
      }
    }

    if(!check) {
      System.out.println("No");
      System.exit(0);
    }

    for(int i = 0; i < n; i++) {
      for(int j = 0; j < n; j++) {
        if(i == j) continue;
        if(s.charAt(i) == s.charAt(j)) {
          System.out.println("No");
          System.exit(0);
        }
      }
    }

    System.out.println("Yes");
  }
}
